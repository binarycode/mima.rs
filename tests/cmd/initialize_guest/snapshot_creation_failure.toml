args = 'initialize-guest zero'
status.code = 1
