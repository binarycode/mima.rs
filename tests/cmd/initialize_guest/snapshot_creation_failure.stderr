error: Failed to run 'ssh -o BatchMode=yes -o ConnectTimeout=1 -o ForwardAgent=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null root@10.210.0.13 qemu-img snapshot -croot /mima/zero-sda.qcow2'

stdout:
foobar

